src/reducers/index.js

```diff
@@ -97,6 +115,21 @@ export default combineReducers({
       }
     }
 
+    if (type === UPDATE_USER_SETTINGS_PENDING) {
```

Comment:
Any objection to putting the PENDING cases and the error cases each in their own switch statements, instead of the ifs?